components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -622,7 +623,10 @@ protected void concludeFlow(HttpServletRequest request, HttpServletResponse resp
                         authenticationResult.getSubject().getUserName(), sessionContextKey,
                         authenticationResult.getSubject().getTenantDomain(), FrameworkUtils.getCorrelation(),
                         createdTimeMillis, sessionContext.isRememberMe());
-                setAuthCookie(request, response, context, sessionKey, applicationTenantDomain);
+                if (request.getAttribute(SKIP_SET_COMMONAUTH_COOKIE) == null
```

Comment:
shall we use something like skipSessionCreation, or skipWebSession? which will make more sense to outsider, and will be more generic and can be used to skip any other such web session specific things

Reply:
also, let's have the positive form in configs to avoid double negation.